.github/workflows/test-fixtures.yml

```diff
@@ -93,6 +94,8 @@ jobs:
             command: --bump --unreleased --with-tag-message "Some text"
           - fixtures-name: test-from-context
             command: --from-context context.json
+          - fixtures-name: test-from-context-does-not-discard-fields
+            command: --context | git cliff --from-context -
```

Comment:
I like this, but I think here we are not comparing the final output against the `git cliff --context`, instead we are asserting what we get from `--from-context`.

It would be better if we add a `context.json` file to the fixture and use that instead of running `--context`. What do you think?

Reply:
> It would be better if we add a context.json file to the fixture and use that instead of running --context. What do you think?

Yes that's great, it would make the test case deterministic instead of depending on output from `--context`. I will fix that.